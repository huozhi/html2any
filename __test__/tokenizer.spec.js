const tokenizer = require('../src/tokenizer')
const html1 = require('./mock/html1')

console.log(tokenizer(html1))
