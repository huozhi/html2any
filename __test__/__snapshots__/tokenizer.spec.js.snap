// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tokenizer parse correctly 1`] = `
Array [
  Object {
    "attribues": Object {},
    "children": null,
    "name": "div",
    "type": "start",
  },
  Object {
    "attribues": Object {},
    "children": null,
    "name": "hr",
    "type": "self-close",
  },
  Object {
    "attribues": Object {
      "data-index": "2",
      "src": "a.jpg",
    },
    "children": null,
    "name": "img",
    "type": "self-close",
  },
  Object {
    "attribues": Object {
      "id": "x34",
    },
    "children": "1234 &amp;",
    "name": "p",
    "type": "start",
  },
  Object {
    "name": "p",
    "type": "end",
  },
  Object {
    "attribues": Object {
      "class": "d3 d2",
      "id": "container",
    },
    "children": null,
    "name": "div",
    "type": "start",
  },
  Object {
    "attribues": Object {
      "disabled": true,
      "value": "123",
    },
    "children": null,
    "name": "input",
    "type": "self-close",
  },
  Object {
    "name": "div",
    "type": "end",
  },
  Object {
    "name": "div",
    "type": "end",
  },
]
`;
